src/homework2_task6_extra.py

```diff
@@ -0,0 +1,11 @@
+'''
+Определите, является ли число палиндромом (читается слева направо и справа налево одинаково).  Число положительное целое, произвольной длинны.
+'''
+#variables
+num = int(input('Enter number '))
+#logic
+if str(num) == str(num)[::-1]:
```

Comment:
Please try to implement it without strings.

Reply:
Done.